canonical/vanilla-framework
# Add p-media-container class

## Done

- Add p-media-container class which adds .5rem of spacing above media elements

Fixes https://github.com/canonical/vanilla-framework/issues/4791

## QA

- Open [demo](https://vanilla-framework-5012.demos.haus/docs/patterns/images)
- See that the image in the 'Media element with spacing' section is wrapped in a container that adds 0.5rem spacing to the top
- Check the wording of the documentation

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

![image](https://github.com/canonical/vanilla-framework/assets/58276363/fd1cc07e-f863-45a3-ac1b-36dccf888aad)
